Gruntfile.js

```diff
@@ -79,7 +79,7 @@ module.exports = function (grunt) {
         uglify: _.extend({
             options: {
                 compress: {
-                    drop_console: true
+                    drop_console: false
```

Comment:
Undo this change. Console calls should be removed from production code. `$log.log` won't, though